src/Toolkit/composer.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "symfony/ux-toolkit",
+  "type": "symfony-bundle",
+  "description": "Twig toolkit for Symfony",
+  "keywords": [
+    "symfony-ux",
+    "twig",
+    "components"
+  ],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    },
+    {
+      "name": "Hugo Alliaume",
+      "email": "hugo@alliau.me"
+    },
+    {
+      "name": "Jean-François Lépine",
+      "email": "lepinejeanfrancois@gmail.com"
+    },
+    {
+      "name": "Simon André",
+      "email": "smn.andre@gmail.com"
+    }
+  ],
+  "require": {
+    "php": ">=8.2",
+    "tales-from-a-dev/twig-tailwind-extra": "^0.3.0",
+    "twig/extra-bundle": "^2.12|^3.0",
```

```suggestion
    "twig/extra-bundle": "^3.19|^4.0",
```

(won't do it on every line, so the idea is: new bundle -> let's start with ideal stack, and see afterwards how hard it is to bring compat with older.. but at least we can ignore all depreacted / legacy / abandonned versions of deps)